CMakeLists.txt

```diff
@@ -323,7 +323,11 @@ if(LIB_INSTALL_DIR)
 endif()
 
 if (NOT EXISTS "${MBEDTLS_FRAMEWORK_DIR}/CMakeLists.txt")
-    message(FATAL_ERROR "${MBEDTLS_FRAMEWORK_DIR}/CMakeLists.txt not found. Run `git submodule update --init` from the source tree to fetch the submodule contents.")
```

Comment:
Why remove information about the submodule though? [It was useful for people who don't use release archives](https://github.com/Mbed-TLS/mbedtls/pull/9809#discussion_r1875005637).